com.unity.render-pipelines.high-definition/Runtime/Lighting/GlobalIllumination.cs

```diff
@@ -98,6 +98,12 @@ public int filterRadius
         [Tooltip("Defines the layers that GI should include.")]
         public LayerMaskParameter layerMask = new LayerMaskParameter(-1);
 
+        /// <summary>
+        /// Defines the LOD Bias for sampling all the textures.
+        /// </summary>
+        [Tooltip("Defines the LOD Bias for sampling all the textures. A higher value will increase peformance and make denoising easier but may reduce visual fidelity.")]
```

Comment:
Sets the LOD Bias HDRP applies to textures in the reflection. A higher value increases performance and makes denoising easier, but it might reduce visual fidelity.